frontend/app/routes/$lang/_protected/applications/$id.tsx

```diff
@@ -39,14 +41,127 @@ export async function loader({ context: { session }, params, request }: LoaderFu
   }
 
   //prevent users from entering any ID in the URL and seeing other users' applications
-  const applications: { id: string | undefined; submittedOn: string | undefined; status: string | undefined; confirmationCode: string | undefined; data: object[] }[] | undefined = session.get('applications');
-  const viewApplication = applications?.find((applications) => applications.id === params.id);
-  if (!viewApplication) {
+  const applications:
```

Comment:
I feel like this type should be exported from `application-history-service-schemas`. Its declaration in this `loader(..)` clutters a lot of the functionality in my opinion.

Reply:
Fixed: b41c8079f3f88d4afce1b05534871d24cab7bd5a